### PR TITLE
Add message class

### PR DESCRIPTION
**Pull request by saralein**
Create message class for response (will eventually be added to requests)
Delete response serializer (message class handles previous responsibility)

### CODE REVIEW

File: server/src/main/java/com/saralein/server/response/Response.java [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
 import com.saralein.server.exchange.Header;
+import com.saralein.server.exchange.Message;
 
 public class Response {
     private final Header header;
     private final byte[] body;
+    private final Message message;
 
-    public Response(Header header, byte[] body) {
+    public Response(Header header, byte[] body, Message message) {
```
Comment (damonkelley) [inline server/src/main/java/com/saralein/server/response/Response.java:11]: I don't think I understand this. What is the `Message` here? How is that related to `header` and `body`?

Comment (saralein) [reply server/src/main/java/com/saralein/server/response/Response.java:11]: After our conversation, going to close this PR.
